packages/client/public/deploy-config.js

```diff
@@ -26,5 +26,10 @@ window.APP_CONFIG.featureFlags = {
 };
 
 // Setting a GOOGLE_TAG_ID enables Google Analytics.
-// Empty for the dev configuration since there is not yet a Google Analytics property for dev.
+//  Empty for the dev configuration by default.
+//  Set to 'G-D5DFR7BN0N' to enable Google Analytics in dev environments.
 window.APP_CONFIG.GOOGLE_TAG_ID = '';
```

Comment:
I chose to keep the empty string default value rather than defaulting to the new GA4 property. I updated the preceding comment instead, which I hope is sufficient for pointing developers towards the correct value to use. 

The primary reason for this decision is to avoid cluttering the console. The GA4 cookie domain is set to our Staging domain, which won't match in a development environment, meaning that anyone whose browser blocks third-party cookies will see `Third-party cookie will be blocked. Learn more in the Issues tab` warnings in their browser's dev console when GA4 attempts to load.